<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CALC</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(240, 243, 249,1)</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>671</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0" columnstretch="0,0,0,0,0" rowminimumheight="0,0,0,0,0,0" columnminimumwidth="0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_leftbracket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_rightbracket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_backspace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>‚Üê</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_allclear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_pow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>x^y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_add">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_sqrt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>‚àö</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_subtract">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_fact">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>!</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_multiply">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_point">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  border-radius: 8px; 
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}


QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}
QPushButton {
  background-color: #f5f5f5;
  border-radius: 8px;
}

#specialButton {
  background-color: #e0e0e0; 
  border: 1px solid #ccc;
}
QPushButton {
	border: 1px solid transparent;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_equal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(0,120,215,255);
  border: none; 
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(20,100,200,255); 
}

QPushButton:pressed{
  background-color: rgba(0,80,180,255);
}

QPushButton:disabled{
  background-color: rgba(100,150,230,255);
}
QPushButton {
  border-radius: 8px; 
}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_divide">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgba(240,240,240,255);
  border: none;
  box-shadow: none;
}

QPushButton:hover{
  background-color: rgba(220,220,220,255);
} 

QPushButton:pressed{
  background-color: rgba(180,180,180,255);
}
QPushButton {
  border-radius: 8px; 
}

QPushButton:disabled{
  background-color: rgba(200,200,200,255);
}</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
  background-color: rgb(255, 255, 255);
  color: rgb(0, 0, 0);
}
QLineEdit {
	border-radius: 8px;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Á±ªÂûã</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Â∏ÆÂä©</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Ê†áÂáÜ</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Â§ßÊï∞</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>‰ΩøÁî®ËØ¥Êòé</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
